go/libraries/doltcore/sqle/index/index_lookup.go

```diff
@@ -460,6 +469,7 @@ type nomsLookupBuilder struct {
 // NewRowIter implements IndexLookup
 func (lb *nomsLookupBuilder) NewRowIter(ctx *sql.Context, part sql.Partition) (sql.RowIter, error) {
 	p := part.(rangePartition)
+	p.nomsRange.Reverse = p.isReverse
```

Comment:
should this be an error if true?

Reply:
My mistake, this line should just be removed entirely, since we're not going to support reverse `IndexTableAccess` on old format.